pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -445,10 +445,11 @@
      *            Number of new partitions of already exist partitioned-topic
      * @param updateLocalTopicOnly
      *            Used by broker for global topic with multiple replicated clusters
-     *
+     * @param force
+     *            Update forcefully without validating existing partitioned topic
      * @returns a future that can be used to track when the partitioned topic is updated
      */
-    void updatePartitionedTopic(String topic, int numPartitions, boolean updateLocalTopicOnly)
+    void updatePartitionedTopic(String topic, int numPartitions, boolean updateLocalTopicOnly, boolean force)
```

Comment:
it looks like we are dropping an API method.

we should keep the existing method, and add a new one that falls back to the new method, with force=false